.gitignore

```diff
@@ -18,3 +18,4 @@ website/node_modules
 *.bak
 *~
 .*.swp
+.idea/
```

Comment:
You're right. It's a bad habit that snuck in when I looked in another direction. Thanks, I'll revert the file.